examples/pipeline_wavernn/functional.py

```diff
@@ -0,0 +1,30 @@
+import torch
```

Comment:
For the purpose of this example, it may just be simpler to have a single file `processing.py` or `signal_processing.py` or `transforms.py` or `transformations.py` combining both `functional.py` and `transform.py`.

Reply:
Good suggestion. I have combined them in `processing.py`